cmd/bosun/expr/expr.go

```diff
@@ -152,6 +152,30 @@ func (s Series) MarshalJSON() ([]byte, error) {
 	return json.Marshal(r)
 }
 
+// sometimes, we prefer not to have the Series in its map format
```

Comment:
Remove these discussion comments. If you would like any comments, make them real godoc-style comments.